### PR TITLE
Feature/reopen closed tab and fuzzy match highlighting

### PR DESCRIPTION
**Pull request by dbuezas**

### CODE REVIEW

**Comment by dbuezas** [comment]
screen shot 2017-08-05 at 19 30 23

File: src/js/actions.js [hunk 1]
```diff
@@ -33,6 +33,12 @@ function closeCurrentTab() {
     });
 }
 
+function reopenClosedTab() {
+  chrome.sessions.getRecentlyClosed( undefined, function([ lastClosed ]) {
+    console.log("lastSessionId", lastClosed.tab.sessionId)
```
Comment (ssundarraj) [inline src/js/actions.js:38]: Please remove the console.log

File: src/manifest.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
   "options_page": "options.html",
   "permissions":[
     "tabs",
-    "storage"
+    "storage",
+    "sessions"
```
Comment (ssundarraj) [inline src/manifest.json:21]: Please add a comma at the end of this line.

File: src/js/actions.js [hunk 3]
```diff
@@ -33,6 +33,12 @@ function closeCurrentTab() {
     });
 }
 
+function reopenClosedTab() {
+  chrome.sessions.getRecentlyClosed( undefined, function([ lastClosed ]) {
```
Comment (ssundarraj) [inline src/js/actions.js:37]: nit: remove space before "undefined" and remove space around "lastClosed".

File: src/js/main.js [hunk 4]
```diff
@@ -153,7 +161,25 @@ function fuzzySearch(){
         populateSearchSuggestions(searchString);
         var fuzz = new Fuse(suggestionList, options);
         var fuzzResult = fuzz.search(searchString);
-        populateSuggestionsBox(fuzzResult);
+        const highlightedResult = fuzzResult.map(({ item, matches }) => {
```
Comment (ssundarraj) [inline src/js/main.js:164]: Please use function(){} instead of fat arrow syntax to maintain consistency.

Comment (ssundarraj) [inline src/js/main.js:164]: nit: Remove spaces after '{' and before '}'.

File: src/js/main.js [hunk 5]
```diff
@@ -153,7 +161,25 @@ function fuzzySearch(){
...
+            const highlightedItem = Object.assign({}, item);
+            matches.forEach(({ key, indices }) => {
```
Comment (ssundarraj) [inline src/js/main.js:166]: nit: Remove spaces after '{' and before '}'.

**Review by ssundarraj — CHANGES_REQUESTED** [review]
Thanks for the PR! Just have some minor formatting comments.

**Comment by ssundarraj** [comment]
Looks like other PRs have some common changes. Would you mind if I merge this first then take a look at the others?

**Comment by dbuezas** [comment]
Hey Ssundarraj, I am used to a particular code formatting so I missed your format a couple of times.
Every pull request builds upon the previous one, so ups... Definitely not ideal, but I wanted to get the functionality I needed quickly. I'll apply your corrections and rebase the other branches

**Comment by dbuezas** [comment]
this is the most recent branch btw
https://github.com/ssundarraj/commander/pull/16

File: src/manifest.json (hunk 2 above)
Comment (dbuezas) [reply src/manifest.json:21]: This is a JSON file, won't work

File: src/js/main.js (hunk 4 above)
Comment (dbuezas) [reply src/js/main.js:164]: sure you don't want to use fat arrow for callbacks? it is pretty standard now a days. There is no "this" context in these functions so it doesn't change anything, but the fat arrow gives very clear visual queues there.

Comment (dbuezas) [reply src/js/main.js:164]: I am following the airbnb linting conventions there, without spaces it is harder to see when the parameters start and end, sure about it?

**Comment by ssundarraj** [comment]
So can you incorporate the review comments in #16. You can also close other PRs since it has all changes. Once your done with that I can review and merge #16. Sounds good?

**Comment by dbuezas** [comment]
👍

**Pull request closed without merging**